packager/packager.cc

```diff
@@ -1028,13 +1048,36 @@ struct PackageSpecBodyWalk {
                     return;
                 }
                 info.visibleToTests_ = true;
+            } else if (auto target = ast::cast_tree<ast::Send>(send.getPosArg(0))) {
+                // Constant::* is valid Ruby, and parses as a send of the method * to Constant
+                // so let's take advantage of this to implement wildcards
+                if (target->fun != core::Names::star()) {
+                    if (auto e = ctx.beginError(target->loc, core::errors::Packager::InvalidConfiguration)) {
+                        e.setHeader("Argument to `{}` must be a constant or the string literal `{}`",
+                                    send.fun.show(ctx), "\"tests\"");
+                    }
+                    return;
+                }
```

Comment:
Can you be sure to test that extra positional/keyword/block arguments here raise errors?


```ruby
# typed: true
A::B::*(0)
A::B::*(x: 0)
A::B::* {}
```